study.md

```diff
@@ -151,7 +153,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+PUT - put would be used when you want to create or update a resource at an exact location on the server
+POST - post would be used when you want to create or update a resource at subordiante location from where you tell it. and have the server handle that info
+DELETE - delete would be used to remove information that is on the server somewhere.
+GET - Get would be used in order to recieve info thats currently on the server somewhere
```

Comment:
Normally we can simplify it to POST is creating a new resource, PUT and PATCH are used to update or modify existing resources.